lib/curl-transport.js

```diff
@@ -54,6 +54,9 @@ function parseStats(stderr) {
 
 
 exports.run = function (command, cb) {
+  if(/;|&|`|\$|\(|\)|\|\||\||!|>|<|\?|\${/g.test(command)) {
```

Comment:
The regexp isn't easily readable and so maintainable and especially two parts of the expression.
The first part is |\|\||\| and the second is |\${.

Reply:
The regexp looks similar to another contribution. 🤔

Reply:
Good Point, that regex was way too complicated and un-necessary. I've changed to only block the necessary special characters needed to break an initial command. 

I do often take from other PRs and patches. This was taken from CVE-2019-10788 and CVE-2019-10787